specification/maps/data-plane/Creator/preview/2023-03-01-preview/features.json

```diff
@@ -120,7 +119,7 @@
       "description": "A valid `GeoJSON Feature` object type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.2) for details.",
       "x-ms-parameter-location": "client",
       "schema": {
-        "$ref": "#/definitions/Feature"
+        "$ref": "#/definitions/FeatureRequestBody"
```

Comment:
Nit: I'd drop "Body". You wouldn't want to see this in an SDK model, right? `FeatureRequest` seems reasonable.

Reply:
Thanks for the feedback Heath! Since we're shipping soon, we can proceed with it as is. Please assist us in merging when you have the opportunity.

Reply:
Since I was already making changes for the naming of patch and put APIs, I went back and fixed this as well. Thanks again!